tests/testqPrefDisplay.cpp

```diff
@@ -7,6 +7,7 @@
 
 #include <QDate>
 #include <QTest>
+#include <QSignalSpy>
```

Comment:
that seems unrelated to this commit?
in general - a commit like this in the middle of this series is understandable, but adds a lot of noise... and makes it harder to review things because I need to make sure that with all the simple renaming there isn't something else in there that slipped through...